config.example.yaml

```diff
@@ -213,6 +225,9 @@ services:
       salt: secret
       default_page_size: 250
       user_agent: RESTBase
+      internal_request_whitelist:
```

Comment:
I am actually wondering if we should move this to the security object, as it is closely tied to the mediawiki_auth cookie scheme. @Pchelolo  @d00rman @eevans, any thoughts?

In any case, we can do that in a follow-up as well.